AWS_FOR_FLUENT_BIT_VERSION

```diff
@@ -1 +1 @@
-2.28.4
+2.29.0
```

Comment:
The new version should be 2.30.0 I think, since there are new features on top of 1.9.10 in here.

Reply:
Please update commit message as well